raiden/ui/cli.py

```diff
@@ -221,6 +222,12 @@ def options(func: Callable) -> Callable:
             show_default=True,
             type=click.FloatRange(min=0.1),
         ),
+        option(
+            "--claims-file-path",
+            type=click.Path(exists=True, dir_okay=False, resolve_path=True),
+            help="Path to a claims file",
+            default="claims.jsonl",
```

Comment:
Don't we need `required=False` here? or remove the default? At least one unit test is failing because the file doesn't exist.

Reply:
Hm yeah good question. Probably removing the default is better.